src/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+
+//import pages
+import Home from './pages/Home';
+import About from './pages/About';
+import SingleCocktail from './pages/SingleCocktail';
+import Error from './pages/Error';
+
+//import componenets
```

Comment:
You don't really this comment. The line you're explaining is quite self-explanatory